.github/workflows/connectors_tests.yml

```diff
@@ -67,7 +72,7 @@ jobs:
           sentry_dsn: ${{ secrets.SENTRY_AIRBYTE_CI_DSN }}
           git_branch: ${{ steps.extract_branch.outputs.branch }}
           git_revision: ${{ steps.fetch_last_commit_id_pr.outputs.commit_id }}
-          github_token: ${{ secrets.GITHUB_TOKEN }}
+          github_token: ${{ env.PAT }}
```

Comment:
I don't have any memory of this being the right or wrong syntax for the output of this.  is this just getting the variable that we export in the find_a_pat script?

Reply:
yep it's arbitrary: https://github.com/airbytehq/airbyte/blob/4ec9cc4d69e1893a82a4ff88b1ec1a6f0ea3201d/tools/bin/find_non_rate_limited_PAT#L45